projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss

```diff
@@ -27,7 +27,7 @@
     padding-left: 3px;
     padding-right: 3px;
     background: #f5e9f2;
-    background-image: linear-gradient(#ecd3e6 2px, transparent 0), linear-gradient(90deg, #ecd3e6 2px, transparent 0), linear-gradient(#ecd3e6 1px, transparent 0), linear-gradient(90deg, #ecd3e6 1px, transparent 0);
+    background-image: linear-gradient(#ecd3e6 0.5px, transparent 0), linear-gradient(90deg, #ecd3e6 1px, transparent 0), linear-gradient(#ecd3e6 0.2px, transparent 0), linear-gradient(90deg, #ecd3e6 0.5px, transparent 0);
```

Comment:
typically using decimals  for `px` values is an anti-patern. If we really need to be that granular, we should use `rem`

Reply:
changed px to pt

Reply:
Resolved this with latest commit